docs/release-process.md

```diff
@@ -0,0 +1,101 @@
+# sambacc Release Process
+
+### Preparation
```

Comment:
Why three `###` (instead of two)? Same holds for the other sections.

Reply:
No particular reason. Changed.